plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts

```diff
@@ -104,6 +104,29 @@ export function transformComplexMediaQuery(mediaQuery: MediaQuery, replacements:
 
 		const replacement = replacements.get(name);
 		if (replacement) {
+
+			if (replacement.truthy.length === 1 && mediaQueryIsSimple(replacement.truthy[0])) {
+				let mediaFeature: MediaFeature | null = null;
+				const replacementMediaQuery = replacement.truthy[0];
```

Comment:
[nit] Should this use destructuring?

Reply:
I liked the parallel with the condition : `(replacement.truthy.length === 1 && mediaQueryIsSimple(replacement.truthy[0]))`